TESTING.md

```diff
@@ -311,18 +320,26 @@ terraform tool through a shell script, and for every test we run
 * `terraform plan -detailed-exitcode` (for ensuring that `plan` is empty right after `apply`)
 * `terraform destroy -auto-approve`
 
-The test runs from GNUMakefile, using
+The test for Cloud Director runs from GNUMakefile, using:
 
 ```bash
 make test-binary
 ```
 
+Running **Tenant Manager** binary tests, using:
+
+```bash
+make testtm-binary
+```
+
 All the tests run unattended, stopping only if there is an error.
 
-It is possible to customise running of the binary tests by preparing them and then running the test script from the `tests-artifacts` directory:
+It is possible to customise running of the binary tests by preparing them and then running the test
+script from the `tests-artifacts` directory. For **Cloud Director** the first command is `make
+test-binary-prepare`, while for **Tenant Manager**, it is `testtm-binary-prepare`:
```

Comment:
I'm suggesting to split this into two pieces and then remove paragraph slightly below. Would look something like this:

> script from the `tests-artifacts` directory. For **Cloud Director** the first command is:
> 
> ```make
> test-binary-prepare
> ```
> 
> while for **Tenant Manager** it is:
> 
> ```make
> testtm-binary-prepare
> ```
>

Reply:
Changed.